### PR TITLE
integrating LLMService & random weighted draw

### PR DESCRIPTION
**Pull request by kfswain**
Fixing groupversion

### CODE REVIEW

**Review by ahg-g — COMMENTED** [review]
I suggest to split the traffic splitting into its own PR

File: pkg/ext-proc/backend/datastore.go [hunk 1]
```diff
@@ -22,3 +23,18 @@ func (ds *K8sDatastore) GetPodIPs() []string {
 	})
 	return ips
 }
+
+func RandomWeightedDraw(models []v1alpha1.TargetModel) string {
+	weights := 0
+	for _, model := range models {
+		weights += model.Weight
+	}
```
[the remark this answers is not in the document]
Comment (kfswain) [reply pkg/ext-proc/backend/datastore.go:31]: The weights don't have to add up to 100 to work properly. Most people probably will. But I don't think we need to enforce that.  Would a log line printing out the sum of the weights suffice?

File: logs.txt [hunk 2]
```diff
@@ -0,0 +1,52 @@
+I1202 23:01:58.268334       1 main.go:67] Listening on ":9002"
```
[the remark this answers is not in the document]
Comment (kfswain) [reply logs.txt]: Done

**Comment by kfswain** [comment]
@ahg-g updated

**Comment by ahg-g** [comment]
/lgtm

can you change the title as well please?

**Pull request merged**